spec/directions/bake_chapter_key_concepts_spec.rb

```diff
@@ -16,7 +18,7 @@
         <div data-type="page">
           <h1 data-type="document-title" id="page1TitleId">Page 1</h1>
           <section id="sectionId1" class="key-concepts">
-            <h3 data-type="document-title">WWF History</h3>
+            <h3 data-type="title">WWF History</h3>
```

Comment:
I switched to using the `RemoveSectionTitle` direction to delete this element, which searches for the first `data-type="title"` element instead of an `h3`. All books I looked at, including the key concepts section in uphysics and calculus, used `data-type="title"`, so I'm hoping the `data-type="document-title"` attribute in these specs is just a typo. But please let me know if you've seen anything different.